server/src/main/java/fr/uga/l3miage/tp4/models/UserEntity.java

```diff
@@ -0,0 +1,24 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+@Getter
+@Setter
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
+public abstract class UserEntity {
```

Comment:
Attention il vous manque l'annotation `@MappedSuperclass`
```suggestion
@Getter
@Setter
@Entity
@MappedSuperclass
@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
public abstract class UserEntity {
```

De plus, un petit retour, votre classe est marquer de `@Entity` , et les classes filles aussi, c'est soit que la classe mère ou la classe fille